scripts/MUST/execute_tool_in_CI.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+TIMEOUT_CMD="/usr/bin/timeout -k 120 120"
+
+# in $(pwd) ist eine datei testcase.c , die ausgewertet werden soll
```

Comment:
If the comment is relevant, we should have it in English instead of German.
Otherwise I would remove it.

Reply:
correct